src/utils/inputValidation/eventProperties.ts

```diff
@@ -70,7 +70,12 @@ export function validateEventProperties(log: ILogger, maybeProperties: any, meth
 export function validateEvaluationOptions(log: ILogger, maybeOptions: any, method: string): SplitIO.EvaluationOptions | undefined {
   if (isObject(maybeOptions)) {
     const properties = validateEventProperties(log, maybeOptions.properties, method).properties;
-    return properties && Object.keys(properties).length > 0 ? { properties } : undefined;
+    let options =  properties && Object.keys(properties).length > 0 ? { properties } : undefined;
+
+    const impressionsDisabled = maybeOptions.impressionsDisabled;
```

Comment:
We would need to refactor the `validateEvaluationOptions` so that the `impressionsDisabled` option is only validated and used for the Evaluator, and for other SDKs it should be sanitized to `false` (or any other falsy value) to ignore the feature.

Eventually, if the feature is included in SDKs, the change can be rolled back.